pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -35,7 +35,7 @@ def _get_output_shape_from_returnn(self,
     limit_dim = None
     if isinstance(limit, Tensor):
       assert limit.is_defined
-      limit_dim = limit.returnn_naming_entry.is_dim
+      limit_dim = limit.returnn_naming_entry.is_size_value.dim_tag
```

Comment:
This is not the same as before. This would crash if `limit.returnn_naming_entry.is_size_value is None`.